### ISSUE TITLE
remove uneeded code from iconcache

### ISSUE THREAD

**Issue by bilelmoussaoui** [bil-elmoussaoui/sni-qt#3]
Iconcache.cpp is still having some old code to remove the cache, which is not needed anymore. I think we should remove that part as it's not needed anymore...
Also, if it's possible to add some debug information about the application id, the icon name, and the full path of it. This will help us fix all the qt4 icons easily  in the future;)
@wa4557 can you take a look at this?

**Comment by andia89** [comment]
Yeah I can do that. Let me have a look at the code. I'm not a good c++ programmer though :)
